### DISCUSSION TITLE
v10.1.0

### DISCUSSION THREAD

**Discussion by curtisdelicata** [familytree365/genealogy#2149] [Announcements]
<hr /><em>This discussion was created from the release <a href='https://github.com/cgdprojects/genealogy/releases/tag/v10.1.0'>v10.1.0</a>.</em>

**Comment by thefourcraft** [comment]
Docker image Installation 🐳 🙏

**Reply by curtisdelicata** [reply]
Does it not work?

**Reply by thefourcraft** [reply]
I didn't know [genealogy](https://github.com/cgdprojects/genealogy) docker support. 
if it does, do you have a tutorial or a readme.md file?

**Reply by curtisdelicata** [reply]
cp .env.example .env;docker-compose build;docker-compose up -d;docker exec app composer install;docker exec app php artisan key:generate for backend

**Reply by thefourcraft** [reply]
I will check it out, thanks 💕 Dockers are amazing 🐳
